bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -56,35 +56,20 @@ spec:
           name: host-var-lib-ovs
         - mountPath: /var/run/openvswitch/
           name: host-var-run-ovs
+        - mountPath: /var/run/ovn-config
+          name: ovn-config-mount
 
         resources:
           requests:
             cpu: 100m
             memory: 300Mi
         env:
         - name: OVN_DAEMONSET_VERSION
-          value: "3"
-        - name: OVN_LOG_NORTHD
-          value: "-vconsole:info"
-        - name: OVN_NET_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: net_cidr
-        - name: OVN_SVC_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: svc_cidr
+          value: "4"
         - name: K8S_NODE
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
-        - name: K8S_APISERVER
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: k8s_apiserver
         - name: OVN_KUBERNETES_NAMESPACE
```

Comment:
@pecameron I think OVN_KUBERNETES_NAMESPACE can be removed too since that is now passed in the config file.

Reply:
OVN_KUBERNETES_NAMESPACE is used elsewhere in the ovnkube.sh script. Also upstream doesn't use operators and we need to be careful to keep this correct.

Reply:
ovnkube.sh ovn-master calls:
ready_to_start_node () {
# See if ep is available ...
  ovn_db_host=$(kubectl --server=${k8s_apiserver} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
    get ep -n ${ovn_kubernetes_namespace} ovnkube-db 2>/dev/null | grep 6642 | sed 's/:/ /' | awk '/ovnkube-db/{ print $2 }')
  if [[ ${ovn_db_host} == "" ]] ; then